charts/nginx/templates/dataplane/nginx-dp-networkpolicy.yaml

```diff
@@ -19,35 +19,19 @@ spec:
     matchLabels:
       component: dp-ingress-controller
       tier: nginx
-      plane: data
       release: {{ .Release.Name }}
   policyTypes:
   - Ingress
   ingress:
-  # Allow traffic from Control Plane
   - from:
-    - podSelector:
-        matchLabels:
-          plane: {{ .Values.global.plane.mode }}
-          component: dp-ingress-controller
-    - podSelector:
-        matchLabels:
-          plane: {{ .Values.global.plane.mode }}
-          component: houston
+    - ipBlock:
+        cidr: 0.0.0.0/0
```

Comment:
So, by this commander pod is allowed to receive traffic from any IP address. Will this affect airgapped customers?

Reply:
It won't invalidate any other firewalls or security group rules such customers have in place. It just means the nginx ingress controller will accept incoming traffic, since the default policy is to deny all ingress traffic I believe (templates/default-deny-network-policy/networkpolicy.yaml). This isn't exposing commander directly, only via the nginx ingress.